rpcs3/rpcs3qt/debugger_frame.cpp

```diff
@@ -531,22 +618,35 @@ void debugger_frame::OnBreakpointList_delete()
 	}
 }
 
-debugger_list::debugger_list(debugger_frame* parent) : QListWidget(parent)
+debugger_list::debugger_list(debugger_frame* parent, std::shared_ptr<gui_settings> settings) : QListWidget(parent)
```

Comment:
Don't pass shared_ptr by value. Use const reference. It looks like this error is a common occurrence in this code (i.e. not only in this file). Would you mind fixing it? Also, I've noticed that it was added by @Megamouse in this commit: https://github.com/RPCS3/rpcs3/commit/ad66dbfd0b4db8cdb1c312e4bd8e08515c290663 (well, unless the settings can libe even after rpcs3_app is destroyed/closed). 

Doesn't seem like there is any need for shared_ptr at all (though in this case the overhead is negligible).

Reply:
@danilaml fyi: pirhoman is going to refactor this file. He has to wait for this Pull request to get merged. So he can also make this change later in favor for a faster merge now.